FirebaseAuth/Sources/AuthProvider/Email/FIREmailPasswordAuthCredential.m

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  */
 
-#import "FIREmailPasswordAuthCredential.h"
+#import "FirebaseAuth/Sources/AuthProvider/Email/FIREmailPasswordAuthCredential.h"
 
-#import "FIRAuthExceptionUtils.h"
-#import "FIREmailAuthProvider.h"
-#import "FIRVerifyAssertionRequest.h"
+#import <FirebaseAuth/FIREmailAuthProvider.h>
```

Comment:
I'm surprised the linter isn't complaining about this. Headers that look like system headers should get their own group, no?

Here and throughout if you choose to take action.